### PR TITLE
feat: 38 add docker environment variable PACT_BROKER_WEBHOOK_HTTP_CODE_SUCCESS

### PR DESCRIPTION
**Pull request by pitschr**
also masks the database password (otherwise it would be visible in e.g. AWS CloudWatch)

### CODE REVIEW

File: pact_broker/config.ru [hunk 1]
```diff
@@ -7,7 +7,14 @@ require_relative 'docker_configuration'
 require_relative 'pact_broker_resource_access_policy'
 
 dc = PactBroker::DockerConfiguration.new(ENV, PactBroker::Configuration.default_configuration)
-dc.pact_broker_environment_variables.each{ |key, value| $logger.info "#{key}=#{value}"}
+dc.pact_broker_environment_variables.each { |key, value|
```
Comment (bethesque) [inline pact_broker/config.ru:10]: Good thinking. You'll find the code to do this is actually in `pact_broker/docker_configuration.rb`

```
def pact_broker_environment_variables
      pact_broker_environment_variable_names.sort.each_with_object({}) do | name, hash |
        hash[name] = name =~ /password/i ? "*****" : @env[name]
      end
    end
```

Would you mind updating that instead? Maybe just put it in a separate PR so I can merge the rest of it, which looks 👍🏽

**Review by bethesque — CHANGES_REQUESTED** [review]

**Comment by pitschr** [comment]
Beth, it looks like the pact-broker-docker  is not checking the specs otherwise the issue which has been fixed here (line 222 and 227): https://github.com/pact-foundation/pact-broker-docker/pull/39/commits/590e284f1742c01dc575c745ee6eab69d83df42f would have been detected earlier. I just realized it because I added a test case for the PACT_BROKER_DATABASE_URL

**Comment by bethesque** [comment]
Good point. We've recently moved from Travis to Github Actions, and while the tests do run in the "Update" workflow, I had forgotten to move the "test on push/pull request" over. I've just added that back now.

**Comment by bethesque** [comment]
And it's out! I did a quick sanity check, but would love to get a 👍🏽 from you if you're using it in prod yourself.

**Pull request merged**